### DISCUSSION TITLE
Mapping slack users to github users for visibility of honk notifications

### DISCUSSION THREAD

**Discussion by sfildis-gc** [oneroar/discussions#10] [Feature Requests]
HEYA

Is it possible to map slack users to github users so when we get messages like:

`sfildis-gc commented on the PR`

we would get 

`@selin commented on the PR` 

and be notified for it instead? :)

**Comment by DragosDumitrache** [comment]
Would a custom setting on your Honk profile where you link the 2 manually work for now? This is because, unless people have a public email address that matches the slack email, I have no information on which I can base the link
